### PR TITLE
PB-266: Added ADR about drawing text and icon size

### PR DESCRIPTION
**Pull request by ltshb**
Based on discussion with @boecklic  and @pakb 

Work is done in #641 

[Test link](https://sys-map.dev.bgdi.ch/preview/feat-pb-114-icon-size/index.html)

### CODE REVIEW

**Review by pakb — APPROVED** [review]
Would it be possible to use a fraction for the scale (e.g. `1 / 3` so that the final pixel value would be a round `24px` or `48px` (without the rounding issue)?

**Comment by ltshb** [comment]
@pakb I updated the adr about icon size based on some finding how icon size/scale is set by openlayer

**Review by pakb — APPROVED** [review]

File: adr/2024_01_31_drawing_icon_size.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+NOTE: The actual size of the icons on the backend `service-icons` is 96px but they are reduced to 48px for the scale 1x !
+
+## Descision
+
+Because the 4th UI size has been added to have bigger icons and not smaller icon it has been decided by Brice Schaffner and Pascal Barth to change the UI sizes as follow:
+
+- small => 24px
+- medium => 36px
+- big => 48px
+- bigger => 72px
```
Comment (boecklic) [inline adr/2024_01_31_drawing_icon_size.md:71]: `large` instead of `bigger`? or `small` `medium` `large`, `xlarge`?

**Review by boecklic — APPROVED** [review]
thanks for clarifying this!

File: adr/2024_01_31_drawing_icon_size.md (hunk 1 above)
Comment (ltshb) [reply adr/2024_01_31_drawing_icon_size.md:71]: Good idea

**Review by hansmannj — APPROVED** [review]

**Comment by hansmannj** [comment]
Thanks for the details :+1:
And +1 for small, medium, large and xlarge

**Review by ltkum — APPROVED** [review]
I also approve the small, medium, large, xlarge idea

File: adr/2024_01_31_drawing_icon_size.md (hunk 1 above)
Comment (ltshb) [reply adr/2024_01_31_drawing_icon_size.md:71]: @boecklic I updated the translation for the size see bottom of https://docs.google.com/spreadsheets/d/1bRzdX2zwN2VG7LWEdlscrP-wGlp7O46nvrXkQNnFvVY/edit#gid=54811248 or search for `extra_large`, what do you think ?

**Pull request merged**